src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -315,13 +315,12 @@ If collected, blood measurements of radioactivity are be stored in
 [Tabular files](../02-common-principles.md#tabular-files) and located in
 the `pet/` directory along with the corresponding PET data.
 
-The OPTIONAL `recording` entity is used to distinguish sampling methods.
+The REQUIRED `recording` entity is used to distinguish sampling methods.
 For example, if an autosampler is used to record continuous blood samples,
 and manual measurements are also taken,
 then the files may have recording labels `autosampler` and `manual`,
 respectively.
-If multiple recording methods are used on the same PET acquisition,
-the `recording` entity MUST be used to distinguish them.
+if the sampling method is unknown, then recording is assumed `manual`.
```

Comment:
```suggestion
If the sampling method is unknown, then recording SHOULD be set as `manual`.
```

Hopefully I'm interpreting this correctly. Saying that recording is "assumed `manual`" implies, to me at least, that the recording entity would be absent and the sampling method would be assumed to be manual, much like how MRI files without the `part` entity are assumed to reflect magnitude values.